src/shared/bus-unit-util.c

```diff
@@ -106,7 +107,18 @@ DEFINE_BUS_APPEND_PARSE("i", ioprio_class_from_string);
 DEFINE_BUS_APPEND_PARSE("i", ip_tos_from_string);
 DEFINE_BUS_APPEND_PARSE("i", log_facility_unshifted_from_string);
 DEFINE_BUS_APPEND_PARSE("i", log_level_from_string);
-DEFINE_BUS_APPEND_PARSE("i", parse_errno);
+
+static int parse_errno_or_action(const char *p) {
+        if (streq(p, "kill"))
+                return EXEC_SYSCALL_ERROR_NUMBER_KILL;
+        else if (streq(p, "log"))
+                return EXEC_SYSCALL_ERROR_NUMBER_LOG;
+        else {
```

Comment:
i'd drop both `else` words, they are unnecessary, since the branches exit immediately